src/editor/dag.rs

```diff
@@ -2080,5 +2105,25 @@ mod tests {
                 Path::from_vec(vec![0]),
             );
         }
+
+        /// this is a regression test for pull request #81.
+        #[test]
+        fn undo_add_child_crash() {
```

Comment:
We should probably add a prefix to this name (the other regression tests start `i_<issue number>`), so I think `pr_81_undo_add_child_crash` would be the most consistent.